### PR TITLE
WIP

### PR DESCRIPTION
**Pull request by ACatThatPrograms**

### CODE REVIEW

File: src/pages/quickstart-steps/FundWallet.jsx [hunk 1]
```diff
@@ -28,15 +30,15 @@ export default function FundWallet({ nextStep }) {
...
         try {
-            let res = await fundAddress(address);
+            let res = await get(siteConfig.customFields.TEST_FUND_API + '/fund/' + address);
             if (res.error) {
                 setError(res.error);
             }
             await updateBalance(ctx, address);
             setLoading(false);
         } catch (exception) {
+            console.log(exception);
```
Comment (adrianbona) [inline src/pages/quickstart-steps/FundWallet.jsx]: Careful Mr. Cat

Comment (ACatThatPrograms) [reply src/pages/quickstart-steps/FundWallet.jsx]: Man, did you see how many I removed though? This is a small price to pay.

**Review by everton0991 — APPROVED** [review]
Looks good!

File: src/adapter/MadNetAdapter.js [hunk 2]
```diff
@@ -72,25 +65,21 @@ class MadNetAdapter {
...
-     */
-    updateContext(reactMadNetContext) {
-        this.context = reactMadNetContext;
     }
 
-    getMadNetWalletInstance() {
-        return this.wallet();
+    init(siteConfig) {
+        if (this.initiated) { return }
+        console.log(siteConfig);
```
Comment (everton0991) [inline src/adapter/MadNetAdapter.js]: Console log here

File: src/adapter/MadNetAdapter.js [hunk 3]
```diff
@@ -72,25 +65,21 @@ class MadNetAdapter {
...
+        this.walletInstance = new MadWallet(false, siteConfig.customFields.MADNET_API);
+        this.provider = siteConfig.customFields.MADNET_API;
+        console.log(this.wallet())
```
Comment (everton0991) [inline src/adapter/MadNetAdapter.js]: Console log here

**Pull request merged**